src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -2083,7 +2135,7 @@ private static void UpdateElementsWithDeprecatedNamespaces(IEnumerable<XElement>
         }
 
         /// <summary>
-        /// Determine if the whitespace preceding a node is appropriate for its depth level.
+        /// Determine if the whitespace preceding a comment is appropriate for its depth level.
```

Comment:
I think the word `node` is more correct here than `comment`, right?. This repeats in some doc comments later. Overzealous Find/Replace?

Reply:
Fixed.